src/IBM.WatsonDeveloperCloud/Sockets/AWatsonWSClient.cs

```diff
@@ -0,0 +1,153 @@
+﻿/**
```

Comment:
Don't use `Watson` in the class name. I think `WebSocketClient` it's a better name